src/robust.jl

```diff
@@ -92,10 +92,11 @@ function winsor!(x::AbstractVector; prop::Real=0.0, count::Integer=0)
         0 <= count < n/2 || throw(ArgumentError("count must satisfy 0 ≤ count < length(x)/2."))
     end
 
-    partialsort!(x, (n-count+1):n)
-    partialsort!(x, 1:count)
-    x[1:count] .= x[count+1]
-    x[n-count+1:end] .= x[n-count]
+    ix = partialsortperm(x, 1:(count+1))
```

Comment:
Same remark here about using `partialsortperm!`.

Reply:
fixed as for trim!